Neos.Media.Browser/Classes/Controller/AssetController.php

```diff
@@ -436,7 +436,8 @@ public function variantsAction(string $assetSourceIdentifier, string $assetProxy
                 'imageProfiles' => $this->imageProfilesConfiguration,
                 'overviewAction' => $overviewAction,
                 'originalInformation' => (new ImageMapper($asset))->getMappingResult(),
-                'variantsInformation' => $variantInformation
+                'variantsInformation' => $variantInformation,
+                'useParentRequest' => !$this->request->isMainRequest()
```

Comment:
Oh dear, that controller code gets out of hand o.O
But that's not your fault of course, the fix seems reasonable!
Just one suggestion:
"use parent request" is a bit cryptic/imperative. Should we call this view variable what it _is_ rather than what the view should _do_ with it, for example:

```suggestion
                'isSubRequest' => !$this->request->isMainRequest()
```